### PR TITLE
Burndown chart - New Sprint

### PR DESCRIPTION
**Pull request by Ana06**
There is still work to do, but If I wait until everything is completely finished before sending a PR it would huge and difficult to review and this can be merged independently. :bowtie: 

I add Trollolo configuration. I am using a version from Github, when I am completely finish I'll release the gem instead and take it from Rubygems, but I already released three times and I always find something else to change :wink: 

I also implemented Sprint feature. A link to start the sprint is shown in the `Sprint#index` page, on the
Sprint which starts today. When clicking the link the Sprint:

- The burndown data file the Sprint will be generated.
- Initial burndown char will be plotted.
- The ploted burndown char will be upload to the first Done column in
  Trello.

Work still to do:

- Program the burndown char update.
- Push burndown char to github.
- Show a hint of whar Start sprint exactly means.
- Release new version of Trollolo.

This will make more efficient our scrum process. It is part of https://github.com/openSUSE/agile-team-dashboard/issues/15

### CODE REVIEW

File: app/controllers/sprints_controller.rb [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Sprints controller
+
+require 'English'
```
Comment (DavidKang) [inline app/controllers/sprints_controller.rb]: For what is this needed?

File: trollolo/.trollolorc [hunk 2]
```diff
@@ -0,0 +1,2 @@
+developer_public_key: ee2c4c43d6008fe6113455bc0fd3a723
```
Comment (DavidKang) [inline trollolo/.trollolorc]: Please remove this file

File: trollolo/burndown-data-01.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (DavidKang) [inline trollolo/burndown-data-01.yaml]: Is this file really needed?

Comment (Ana06) [reply trollolo/burndown-data-01.yaml]: yes, Trollolo need it at the moment. I already thought about the fact that it shouldn't be needed: https://github.com/openSUSE/trollolo/issues/76 Feel free to send a PR :wink:

File: trollolo/.trollolorc (hunk 2 above)
Comment (Ana06) [reply trollolo/.trollolorc]: ouch! I did it again! :sob:

File: app/controllers/sprints_controller.rb (hunk 1 above)
Comment (Ana06) [reply app/controllers/sprints_controller.rb]: To use `$CHILD_STATUS`. This is enforced by [Rubocop](https://rubocop.readthedocs.io/en/latest/cops_style/#stylespecialglobalvars). You may want to read [Ruby style guide](https://github.com/bbatsov/ruby-style-guide#no-cryptic-perlisms). :wink:

Comment (DavidKang) [reply app/controllers/sprints_controller.rb]: Nice!, thanks :smile_cat:

**Comment by Ana06** [comment]
[(3) P19: Frontpage: Burndown Chart](https://trello.com/c/Gm0FGST1/567-3-p19-frontpage-burndown-chart)
